scripts/ci/Makefile

```diff
@@ -18,7 +18,7 @@ export UNAME
 CONTAINER_RUNTIME := docker
 export CONTAINER_RUNTIME
 
-alpine: ZDTM_OPTS=-x zdtm/static/binfmt_misc -x zdtm/static/sched_policy00
+alpine: ZDTM_OPTS=-x zdtm/static/binfmt_misc -x zdtm/static/sched_policy00 -x zdtm/transition/rseq01
```

Comment:
@mihalicyn Could you add `FIXME` comment as well?

Reply:
great idea. Will do.